xbmc/settings/AdvancedSettings.cpp

```diff
@@ -1385,14 +1388,18 @@ void CAdvancedSettings::AddSettingsFile(const std::string &filename)
   m_settingsFiles.push_back(filename);
 }
 
-float CAdvancedSettings::GetLatencyTweak(float refreshrate)
+float CAdvancedSettings::GetLatencyTweak(float refreshrate, bool isHDREnabled)
 {
   float delay = m_videoDefaultLatency;
   for (int i = 0; i < (int) m_videoRefreshLatency.size(); i++)
   {
     RefreshVideoLatency& videolatency = m_videoRefreshLatency[i];
     if (refreshrate >= videolatency.refreshmin && refreshrate <= videolatency.refreshmax)
+    {
       delay = videolatency.delay;
+      if (isHDREnabled && videolatency.hdrlatency != NULL)
```

Comment:
```suggestion
      if (isHDREnabled && (videolatency.hdrlatency > 0.0f || videolatency.hdrlatency < 0.0f))
```
Compare float with NULL is not correct.

Reply:
```suggestion
      if (isHDREnabled && videolatency.hdrlatency != 0.0f)
```
removes the need for two extra checks